scss/_grid.scss

```diff
@@ -63,9 +67,13 @@ Tablet (Smaller than 959px)
 .row                                            { width:768px; }
 .row .row                                       { margin:0 -14px; }
 
+.grid-1, .grid-2, .grid-3, .grid-4,
+.grid-5, .grid-6, .grid-7, .grid-8,
+.grid-9, .grid-10, .grid-11, .grid-12           { margin:0 14px; }
+
 @for $i from 1 through 12                       {
-    .grid-#{$i}                                 { width: #{36 + 64 * ($i - 1)}px; margin:0 14px; }
-    .grid-#{$i}-tablet                          { width: #{36 + 64 * ($i - 1)}px; margin:0 14px; }
```

Comment:
Yer that's it, also i guess you could merge the two together too as there the same code right?